src/components/transactions/TxList/index.tsx

```diff
@@ -29,38 +32,60 @@ export const TxListGrid = ({ children }: { children: (ReactElement | null)[] }):
 
 const TxList = ({ items }: TxListProps): ReactElement => {
   const router = useRouter()
-  // Ensure list always starts with a date label
-  const list = useMemo(() => {
-    const firstDateLabelIndex = items.findIndex(isDateLabel)
-    const firstTxIndex = items.findIndex(isTransactionListItem)
-    const shouldPrependDateLabel =
-      (firstDateLabelIndex === -1 || firstDateLabelIndex > firstTxIndex) && firstTxIndex !== -1
+  const [filter] = useTxFilter()
 
-    if (!shouldPrependDateLabel) {
+  const list = useMemo(() => {
+    if (!filter.type) {
```

Comment:
Good catch, we don't need to insert the date for regular txns anymore, since we don't have old-school pagination now.

Reply:
True. Checking the `filter.type` seems sufficient.